elastic/beats
# metricbeat - fix elasticsearch and kibana integration tests failures in 8.0

### Summary

Rel https://github.com/elastic/beats/pull/28659

The environments used by metricbeat integration tests are bumped to 8.0 and both elasticsearch and kibana packages are currently failing:
- elasticsearch: new fields were added and needs to be documented
- kibana: the integration tests for the `kibana.settings` metricset are failing since the API it hits does not exist as of 8.0. I removed the failing tests to unblock us for now and will follow up with a clean up to remove the `settings` metricset completely from the codebase (TODO create issue)

### Testing
- apply https://github.com/elastic/beats/pull/28659 changes to local branch to execute tests against 8.0
- run `MODULE=elasticsearch mage pythonIntegTest`
- run `MODULE=kibana mage integTest`
- ensure both tests suite are successful
